vrc-get-gui/locales/ja.json5

```
@@ -76,6 +76,7 @@
     "installed {{name}} version {{version}}": "{{name}} バージョン {{version}} をインストールしました。",
     "removed {{name}}": "{{name}} をアンインストールしました。",
     "upgraded all packages": "すべてのパッケージを更新しました。",
+    "applied changes in bulk": "バルクの応用変化。",
```

@Sayamame-beans どうするのがいいかな

---

Yeah, used DeepL as it seem that's what you used as well. I don't speak either Japanese os Chinese.

---

No problem. I think it might be better to not update languages not familiar with in feature pull requests and it might be better to update localization in another PRs. (or using something like crowdin)

---

Will do now

---

更新するのか削除するのかって完全に分かれていたと思う(同時に複数種類の操作が実行されることはなかったはず)ので、メッセージを別々に分けた方が良いような気はします…?

```suggestion
    "applied changes in bulk": "パッケージの一括変更を適用しました。",
```
とかですかね? ちょっとしっくりこないですけれど…

---

en:
I think that updating, adding or deleting multiple packages in ALCOM have been completely separated operations (I don't think there were ever multiple types of operations performed at the same time), so I feel like it would be better to separate the messages...maybe?

---

> it would be better to separate the messages...maybe?

I agree